sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -20,14 +20,16 @@
     AzureFunctionEnvironmentVariable,
     AzureWebAppEnvironmentVariable,
     ContainerAppEnvironmentVariable,
-    KubernetesEnvironmentVariable
+    KubernetesEnvironmentVariable,
+    EMPTY_LABEL
 )
 
 from ._user_agent import USER_AGENT
 
 if TYPE_CHECKING:
     from azure.core.credentials import TokenCredential
 
+JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
```

Comment:
Given that this scenario is read-only - I think this should be Mapping
```suggestion
JSON = Mapping[str, Any]  # pylint: disable=unsubscriptable-object
```